src/components/try-it/copy-button/CopyButton.tsx

```diff
@@ -0,0 +1,13 @@
+"use client";
+
+export default function CopyButton() {
+    function copyRequestURL(event) {
+    	navigator.clipboard.writeText(event.target.closest(".collapse").querySelector(".collapse-title").textContent);
```

Comment:
There is no reason to do it this way. Just have the parent pass down the url using the props.